src/main/kotlin/com/carneirofatima/vehicleterminal/InvalidBodyException.kt

```diff
@@ -0,0 +1,3 @@
+package com.carneirofatima.vehicleterminal
+
+class InvalidBodyException(message: String) : Exception(message)
```

Comment:
Perhaps is better extends `RuntimeException` instead of `Exception`. The reason is because some functionality java-based have differents behaviors if the exception is Runtime or Exception 
For example: Rollback in Transactional contexts.
I wrote a post on medium about Transactions and rollbacks depends on Exceptions: 
https://medium.com/dev-cave/controle-transacional-34b9de948b8d